src/server/config/schema.js

```diff
@@ -81,6 +81,10 @@ module.exports = () => Joi.object({
   })
   .default(),
 
+  ops: Joi.object({
```

Comment:
The suggestion sounds fine to me. 

> i see other monitor configs in the hapi docs that may end up here.

Can you provide some details about these?

> are there any other configs you see being in the ops object

If we rename it to `monitor` then yes, I'm sure we'll add more configs to the object